packages/react/mapbox-resolution/index.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * This empty file is used in our `package.json` to resolve the `mapbox-gl` module. It provides a resolution for
+ * bundlers like Webpack which treats a dynamically imported module like `mapbox-gl` as a separate bundle even if it
+ * isn't used.
+ */
+module.exports = {};
```

Comment:
We can move this directory. Any thoughts on where it should live? Or is the root `react` dir sufficient?

Reply:
Maybe we should put it in a `lib/mapbox-gl` folder?